### PR TITLE
Ports necropolis chest loot: Concussive Gauntlets from Shiptest

### PR DESCRIPTION
**Pull request by GraveHat**
I am going to punch a rock and win.

# Document the changes in your pull request

Ports the concussive gauntlets from Shiptest, the gauntlets allow you to mine without a pickaxe even while holding something in your hand. The gauntlets have been tweaked to protect your arms like the bone bracers because it has actual armor values very much like the bracers and also look like they cover your arms while you're wearing them, also increased the acid armor to 100 so your gamer loot doesn't die to acid when you arrive on station.

Why should this be a thing miners can get? It's a fairly inconsequential item that at most can be used as better arm armor for your glove slot while giving the miners something cool looking to wear.

This has been tested and works as intended, I am open to reverting or changing the tweaks I've made if they are seen as unneeded. I was also going to give them the ability to break basalt in 1 hit and or have a mining scanner effect when worn but I've spent way too long trying to get either to work and I'm stumped on how to implement them.

Leaving suggestions that clean up the code and or update them to fit our codebase would be much appreciated because I do not have the confidence to identify what's entirely wrong with the code itself, porting it in its entirety instead and making it work in our environment.

Edit: Forgot to showcase its raw stats so that's in wiki documentation now.

# Spriting
Mob Sprite
image
With Armor
image
image
Without Armor
image
image
Item Sprite
image
image

# Wiki Documentation

Should be added to the necropolis chest loot list, images provided above.
Stat Block: Also yes this does cover your arms and not your hands.
armor = list(MELEE = 25, BULLET = 25, LASER = 15, ENERGY = 25, BOMB = 100, BIO = 0, RAD = 0, FIRE = 100, ACID = 100)

# Changelog
:cl:  
rscadd: Adds concussive gauntlets to necropolis chest loot
tweak: increased acid armor to 100
tweak: tweaked to covers arms instead of hands
imageadd: added concussive gauntlets sprites
/:cl:

### CODE REVIEW

File: code/modules/clothing/gloves/miscellaneous.dm [hunk 1]
```diff
@@ -187,3 +187,45 @@ obj/effect/proc_holder/swipe
...
+	toolspeed = 0.1
+	strip_delay = 40
+	equip_delay_other = 20
+	body_parts_covered = ARMS
+	cold_protection = ARMS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = ARMS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	resistance_flags = LAVA_PROOF | FIRE_PROOF //they are from lavaland after all
+	armor = list(melee = 25, bullet = 25, laser = 15, energy = 25, bomb = 100, bio = 0, rad = 0, fire = 100, acid = 100)
```
Comment (tattax) [inline code/modules/clothing/gloves/miscellaneous.dm]:
```suggestion
	armor = list(MELEE = 25, BULLET = 25, LASER = 15, ENERGY = 25, BOMB = 100, BIO = 0, RAD = 0, FIRE = 100, ACID = 100)
```

**Review by KillerOrcaCora — APPROVED** [review]
sexy sprite

**Comment by JamieD1** [comment]
wheres the damn conflict, I cant see it

**Comment by GraveHat** [comment]
> wheres the damn conflict, I cant see it

The conflict was because I put the sprite for the item itself inside the same file the ember flowers were in, since that one got merged this pr was using different sprites in that file. What I did to resolve this issue was update my main branch, then go into the artefact.dmi file in VSC with the updated .dmi file with the new flower sprite, switched my branch from master to the one this pr was using with the screen still open which allowed me to save the updated sprites, then it was just a process of readding in the sprite for the concussive gloves.

No idea why it still shows up as conflicted though even though I literally took it from the latest branch :/
I could still move the glove sprites to gloves.dmi If you want though.

**Comment by GraveHat** [comment]
Found out how to fix a conflict.

**Pull request merged**